cupy/linalg/_util.py

```diff
@@ -151,3 +151,19 @@ def _tril(x, k=0):
 def _triu(x, k=0):
     _triu_kernel(k, x)
     return x
+
+
+def batch_identity(batch_shape, n, dtype):
```

Comment:
I noticed `_util` is not a public module, so it's OK that we don't prefix the function name with an underscore.